ci_tests/.dockerignore

```diff
@@ -1,3 +1,2 @@
 **/node_modules
 evm_binding
-solana_binding
```

Comment:
these are / should be copied from the `solana/Dockerfile`. why has this been removed?